src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+4.0.0
+-----
+
+* [BC BREAK] removed `ConsoleExceptionEvent`
```

Comment:
Whatever PR I copied from had it, so that's what I rolled with.  Rebased and removed the prefix.